blockchain/vm/opcodes.go

```diff
@@ -123,6 +123,7 @@ const (
 	MSIZE
 	GAS
 	JUMPDEST
+	MCOPY
```

Comment:
Isn't this 0x5c? I think we need 0x5e

Reply:
We could add filler enums until we have TLOAD (0x5C) and TSTORE (0x5D)
```suggestion
	OP_5C
	OP_5D
	MCOPY
```